internal/gcerr/gcerr.go

```diff
@@ -50,6 +50,9 @@ const (
 
 	// The feature is not implemented.
 	Unimplemented ErrorCode = 6
+
+	// The system was in the wrong state.
+	FailedPrecondition ErrorCode = 7
```

Comment:
Doesn't `gcerr_string.go` need to be updated for this?

Reply:
The `stringer` tool is currently broken in module mode. I'm going to work with the tools team in New York to work around that. I'll address in a followup PR.

Reply:
Add a TODO?

Reply:
Done.